distribution/docs/src/main/resources/_contents/_data/data-intro-contents.adoc

```diff
@@ -211,6 +212,9 @@ Though the following attributes are required on all metacard types, setting thei
 [NOTE]
 ====
 It is highly recommended when referencing a default attribute name to use the `ddf.catalog.data.Metacard` constants whenever possible.
+Mapping the to a neutralized taxonomy allows for higher quality transformations between different
+formats and for improved federation. The neutral profile lets more information be indexed and stored by ${branding}
+for use in querying and transforming.
```

Comment:
Nit: I'd prefer Recommend changing "The neutral profile lets more information be indexed and stored by ${branding} for use in querying and transforming" to "The neutral profile facilitates improved search and discovery across disparate data types."